docs/source/upcoming_release_notes/1164-Add_E1C_to_LODCM.rst

```diff
@@ -0,0 +1,30 @@
+1164 Add E1C to LODCM
+#################
+
+API Changes
+-----------
+- self.E1C now available to move first tower energy only on LODCMs
```

Comment:
Should we rename this section to API breaks? The intent here is to flag things that would need a major release increment. This probably all should have been in the `Device Updates` section.